packages/browser-tests/src/server.ts

```diff
@@ -1,25 +1,57 @@
-import { ChildProcess, exec } from "child_process";
-import * as net from "net";
-import { dirname, join } from "path";
 import { fileURLToPath } from "url";
+import * as path from "path";
+import * as fs from "fs";
 
 import { chromium } from "@playwright/test";
 import cors from "cors";
 import express, { Request, Response } from "express";
+import * as net from "net";
 
 import adminRouter from "./routes/admin.js";
 import { setPage, getPage, closeBrowser } from "./browser/index.js";
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = dirname(__filename);
 
 const app = express();
 
 app.use(cors());
 app.use(express.json());
+
+// Serve built shared API for browser import
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const distRoot = path.resolve(__dirname, "..");
+const sharedDir = path.resolve(distRoot, "shared");
+// Serve bootstrap script from source (not emitted by tsc)
+const assetsDir = path.resolve(distRoot, "..", "src", "assets");
+app.use("/shared", express.static(sharedDir));
+app.use(express.static(assetsDir));
+
+// Resolve installed @waku/sdk version for CDN import map
+let wakuCdnVersion = "0.0.30";
```

Comment:
this is old version already
any why it is not in some kind of configuration file or retrieved from environment?